Foundation/NSStream.swift

```diff
@@ -60,17 +64,17 @@ public func <(lhs: Stream.PropertyKey, rhs: Stream.PropertyKey) -> Bool {
 // NSStream is an abstract class encapsulating the common API to NSInputStream and NSOutputStream.
 // Subclassers of NSInputStream and NSOutputStream must also implement these methods.
 public class Stream: NSObject {
-    
+
     public override init() {
-        
+
     }
     
     public func open() {
-        NSUnimplemented()
+        streamOpen()
```

Comment:
This can be NSRequiresConcreteImplementation and calls can just be simplified to open and close without needing to override an internal method